t/fanotify-common.sh

```diff
@@ -2,7 +2,14 @@
 
 # Check for kernel support and privileges
 fanotify_supported() {
-    ../../src/fsnotifywait --fanotify $* 2>&1 | grep -q 'No files specified'
+    if [ -z "$(grep 'kthreadd' /proc/2/status 2>/dev/null)" ]; then
+        # FIXME: fanotify is broken in containers
+        # https://stackoverflow.com/a/72136877/2995591
```

Comment:
Please be accurate. It is not working on overlayfs.
It would be pretty easy to fix by using tmpfs but I can do that later.